README.md

```diff
@@ -1,3 +1,37 @@
 # KubeRobo
 
+KubeRobo will systematically demonstrate SIG Robotics’ end-to-end cloud-native robotics solution. It will serve as a comprehensive guide to all open source projects under the SIG Robotics umbrella, covering their scope, roadmap, and feature plans.
 
+![Introduce](./docs/images/intro.png)
+
+## Basic
+
+KubeEdge and its ecological components, such as sedna, ianvs, and edgemesh, serve as the technical base for a device-edge-cloud integrated robot solution, managing node equipment, orchestrating applications, network communications, and AI plugins.
+
+## Platform
+
+RoboOMS(Robot Operation Management Service) centrally manages multi-brand heterogeneous robots, supporting robot motion planning, control, and monitoring operation and maintenance.
```

Comment:
(minor, no block) `RoboOMS(Robot Operation Management Service)` needs to be described with the same abbreviation in picture? so the reader can see which component here on the architecture?

Reply:
That make sense.